sbteclipse-core/src/main/scala/com/typesafe/sbteclipse/core/EclipsePlugin.scala

```diff
@@ -225,7 +225,7 @@ trait EclipsePlugin {
 
   object EclipseProjectFlavor extends Enumeration {
 
-    val Scala = Value
+    val ScalaIDE = Value
```

Comment:
I understand wanting to rename it.  In that case, let's make "Scala" be an alias fo rthis one and deprecate Scala.

```scala
@deprecated("Use ScalaIDE", "2.4.0")
val Scala = ScalaIDE
```